docs/packages.md

```diff
@@ -16,6 +16,28 @@ The first package in the Dark package ecosystem is for the Slack API. Use these
 
 ![Slack](assets/packages/slackpackage.png)
 
+### getAllConversations
+
+??
+
+### getConversations
+
+Takes your Slack token as a string and returns a list of dicts containing your public Slack channel names and ids.
+
+### getPrivateConversations
+
+Takes your Slack token as a string and returns a list of dicts containing your private Slack channel names and ids.
+
+### oauth
+
+Takes your Slack clientID (string), clientSecret (string), token datastore (Any), and oauth code (string) and returns a result. 
```

Comment:
`(Any)` is technically correct but I think it'd make sense to say DB or Datastore here, since no other type will work, even if we can't statically check it. (And I think `String` - or are we settling on type names being lowercase? They're not in the UI.)